Trustroots/trustroots
# Use existing RulesText component in Angular modal

#### Proposed Changes

* Use existing RulesText React inside Angular modal
* Remove now redundant Angular template with the same text.
* Remove the extra spacing, seems unnecessary 

#### Testing Instructions

* Open http://localhost:3000/signup
* Click "rules" text and confirm it works
   <img width="1036" alt="Screenshot 2020-11-29 at 20 55 31" src="https://user-images.githubusercontent.com/87168/100550900-e6d9ba80-3285-11eb-9a70-aa29fd9d2dd1.png">

* Open http://localhost:3000/rules and confirm it works too
